app/models/reports/schema_v2.rb

```diff
@@ -3,19 +3,30 @@ class SchemaV2
     include BustCache
     include Memery
 
+    FIELDS = %i[
+      controlled_under_care
+      cumulative_assigned_patients
+      cumulative_registrations
+      lost_to_follow_up
+      missed_visit_lost_to_follow_up
+      missed_visit_under_care
+      monthly_registrations
+      patients_under_care
+      uncontrolled_under_care
+      visited_no_bp_under_care
+      visited_no_bp_lost_to_follow_up
+    ].freeze
```

Comment:
Should this go into `FacilityState`?

Reply:
Also see renames in https://github.com/simpledotorg/simple-server/pull/2754.

Reply:
No don't think so, these are just the fields that Repository cares about from FacilityState.